changelog.txt

```diff
@@ -0,0 +1,3 @@
+feature - Adds support for multiple Realtime Database instance rule deploys via the
```

Comment:
instances rules deploy? (I think both instances and rules should be plural, but don't feel strongly)